src/devices/src/virtio/queue.rs

```diff
@@ -39,11 +39,10 @@ impl fmt::Display for QueueError {
         use self::QueueError::*;
```

Comment:
Could `thiserror::Error` be used on `QueueError` to avoid needing to write this manually?

Reply:
Done in a separate commit.